### ISSUE TITLE
Line 55 should be

### ISSUE THREAD

**Issue by SebiTimeWaster** [jfhbrook/openscad-fillets#1]
module fil_linear_i(l, r, angle=90) {
not
module fil_linear_o(l, r, angle=90) {

but i am still not sure what these forms should be used for...

**Comment by asexto** [comment]
Yes, the typo is still there, I was going to open the same issue!
